packages/base/batch.js

```diff
@@ -0,0 +1,26 @@
+import { Transform } from 'stream'
```

Comment:
I wonder. Are we using `stream` or `readable-stream`? Does it make sense to mix?

That said, what about `through2`?

Reply:
`through2` can be used as well, I used `Transform` after reading [this hint](https://www.npmjs.com/package/through2#do-you-need-this). But I'm not aware of the differences between `stream` and `readable-stream`.